plugin/pkg/admission/nodeenv/admission.go

```diff
@@ -0,0 +1,234 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
Wasn't "All rights reserved" deprecated? I was of the impression that this even fails the tests.